components/shooter.py

```diff
@@ -36,8 +37,10 @@ def setup(self) -> None:
         self.outer_motor.setInverted(True)
         self.centre_motor.setInverted(False)
 
-        self.outer_motor.setNeutralMode(ctre.NeutralMode.Coast)
-        self.centre_motor.setNeutralMode(ctre.NeutralMode.Coast)
+        for motor in self.outer_motor, self.centre_motor:
+            motor.configVelocityMeasurementPeriod(1)
```

Comment:
Does this actually work? I thought this needs to be an enum.

Reply:
Seems to